### PR TITLE
move to prod

### PR DESCRIPTION
**Pull request by pbujnicka**

### CODE REVIEW

File: tutorials/spa-create-automation/spa-create-automation.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ An Automation is a succession of steps to orchestrate multiple activities and ap
 
 [ACCORDION-BEGIN [Step 1: ](Create the Automation)]
 
- In this exercise, you will automate the process to read the *sales order* details from an Excel and select the specific sales order details based on the input from the submitted form. To design your automation, you will need an excel file filled with the sales orders data. You have the possibility to create it yourself using the following data:
+In this exercise, you will automate the process to read the *sales order* details from an Excel and select the specific sales order details based on the input from the submitted form. To design your automation, you will need an excel file filled with the sales orders data. You have the possibility to create it yourself using the following data:
```
Comment (MichaelCzcz) [inline tutorials/spa-create-automation/spa-create-automation.md:29]: Excel

File: tutorials/spa-create-decision/spa-create-decision.md [hunk 2]
```diff
@@ -68,120 +68,143 @@ A data type describes the data structure that can be used as an input and/or out
...
+    - Under Rule Type select **Decision Table**,
+    - In the Rule Name enter **Determine Approver**,
+    - In the Rule Description enter **Rule to identify the potential approvers for sales order**,
+    - Choose **Next Step** button.
 
-6. Under **Result Data Types**, select **Determine Approver Output** as the output or result of the decision table.
+    !![002](017.png)
 
-7. Select **User Group** and **Email** and select **Next Step**.
+    > A decision table is a tabular representation of the rule with If and Then header and row columns. If-header columns contain the expressions which are evaluated, and Then-header columns contain the result structure that will be returned after the decision is run.
```
Comment (MichaelCzcz) [inline tutorials/spa-create-decision/spa-create-decision.md:153]: Missing comma before "which"

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/spa-create-automation/spa-create-automation.md (hunk 1 above)
Comment (pbujnicka) [reply tutorials/spa-create-automation/spa-create-automation.md:29]: done

File: tutorials/spa-create-decision/spa-create-decision.md (hunk 2 above)
Comment (pbujnicka) [reply tutorials/spa-create-decision/spa-create-decision.md]: done

**Pull request merged**